docs/howto/toy.rst

```diff
@@ -146,6 +146,8 @@ x86_64, but you could as well use AVR or riscv here.
 
 .. code:: python
 
+    from ppci.irutils import Verifier
+    from ppci import api
     Verifier().verify(ir_module)
```

Comment:
Now that I look at this, I realize this could also be written as:
```python
api.verify_module(ir_module)
```
which removes the need to import the `Verifier` class.

Reply:
yes I do, thank you for the great tutorial